### ISSUE TITLE
Green Balloon Alert Issues

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#364]
**BASE Testing March 9th**
Safari 11.0.3
Mac os 10.13.3

**Green Balloon Alerts**
See [Two-balloon design doc](https://docs.google.com/document/d/1LrOWT4RZiS4emrumW479POK--zWveWiylZMd7unNyi0/edit#heading=h.uflz7g8a7u0)

There are a few issues with green balloon alerts. In general, I am just getting too much information.

1. Upon removal, I hear  information about the Green Balloon's position of where it was.
**Solution for removal of green balloon:**
- [x] Upon removal, the user should only ever hear, "Green balloon removed from Play Area." Nothing else.

2. Things are more complicated, of course, when adding the Green Balloon back after interaction has occurred. However, the goal of the alert is to let the user know 3 things: 1. they have added the green balloon back, 2. where it is, and 3. in special cases, if it is moving. We can leave all other details such as level of charge, induced charge for the user to find out for themselves through deeper investigation when they actually interact with the Green Balloon.
**Solution when adding Green Balloon back:**
- [ ] For a green balloon that is not moving and is added back to the Play Area, restrict alert to "Green balloon added. [+ Sticking state & location].
- [ ] For the special case where the green balloon is added and is moving because Play Area has changed (e.g. was at wall when removed, but wall is now removed when it is added back), the alert pattern should follow the release and independent balloon movement patterns (OPTION B). See [Grid design doc](https://docs.google.com/document/d/1WNVq5_-Hc9LSdeZVSn2edlwYwpHtGNT1ThFD9NXpZ5Q/edit#heading=h.4xn2gqwfg1t). Sorry, this was not indicated in the two-balloon design document. I have updated the [Green Balloon Alerts section](https://docs.google.com/document/d/1LrOWT4RZiS4emrumW479POK--zWveWiylZMd7unNyi0/edit#heading=h.uflz7g8a7u0) in Two-balloon design doc.
 - Note that the only needed change to the release movement pattern is that the [+ Indication of release] is replaced with "Green balloon added. [+ Release movement 1, 3, and 4]."

**3 and 4 actually have to do with the Wall alert, not the Green Balloon alert.**

3. When I remove the wall, a couple of times I heard something about the hidden green balloon. I am having trouble reproducing this. Perhaps it was a stale alert. If issues 1 & 2 are fixed, we can see if this pops up again.

4. Wen adding the wall back to play area when a balloon is resting at right edge of Play Area, I hear "Green balloon moving left." While it is true that the balloon moved left to make room for the wall, it is not important information. 
- Question: Can we suppress the Green balloon's movement alert, or instead just append its location at the wall, e.g., "Wall added to Play Area. Green balloon at upper wall."

**Comment by terracoda** [comment]
@jessegreenberg, please split up issue if you need to and re-assign to me if you have questions.

**Comment by terracoda** [comment]
@jessegreenberg, or alternatively, just re-assign to me to split up the issue.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, no need to separate I can work on this issue.

**Comment by jessegreenberg** [comment]
All of this should be fixed with the above commit, except the following:

> Wen adding the wall back to play area when a balloon is resting at right edge of Play Area, I hear "Green balloon moving left." While it is true that the balloon moved left to make room for the wall, it is not important information.

Ill work on this next.

**Comment by jessegreenberg** [comment]
> Question: Can we suppress the Green balloon's movement alert, or instead just append its location at the wall, e.g., "Wall added to Play Area. Green balloon at upper wall."

Ah, this was actually fixed in the above commit. :)

Closing this issue.
